packages/replay-data/src/analysis/annotateExecutionPoints.ts

```diff
@@ -103,8 +103,9 @@ export async function annotateExecutionPoints(
     const filePath = getFilePath(spec.repository, location.url);
     const lines = fs.readFileSync(filePath, "utf8").split("\n");
     let found = false;
+    const targetLine = location.source.trim();
```

Comment:
NOTE: For some reason, the `source` line is not trimmed anymore. I only fixed this because it caused issues when re-testing `10609`. 🤷‍♀️